### PR TITLE
output hang annotation information in the bhr job

### PR DESCRIPTION
**Pull request by squarewave**
This PR outputs hang annotation information which we are currently largely discarding from the bhr ping. This will be used to display additional hang information in the portal located at https://fqueze.github.io/hang-stats

### CODE REVIEW

**Comment by squarewave** [comment]
@BenWu are you free to take a look at this?

**Comment by squarewave** [comment]
To clarify what testing I've done. I've run this job manually, sending its output to https://analysis-output.telemetry.mozilla.org/bhr/data/hang_aggregates/hangs_main_annotated_current.json. I also have a [PR](https://github.com/fqueze/hang-stats/pull/2) to the portal to consume this information, and it all looks good.

File: mozetl/bhr_collection/bhr_collection.py [hunk 1]
```diff
@@ -334,17 +349,14 @@ def ingest_row(self, row):
                 # If we're below the acceptance threshold, just lump it under (other) below
                 # its parent.
                 last_stack = stack_table.key_to_index(("(other)", lib_name, last_stack))
+                last_cache_item_index = cache_item_index
```
Comment (BenWu) [inline mozetl/bhr_collection/bhr_collection.py:352]: What's this for?  `last_cache_item_index` doesn't seem to be used after the break

**Review by BenWu — APPROVED** [review]
Looks good to me.  Let me know if you're ready to merge

File: mozetl/bhr_collection/bhr_collection.py (hunk 1 above)
Comment (squarewave) [reply mozetl/bhr_collection/bhr_collection.py:352]: Hmm, I think this snuck in, and isn't properly related to my annotations patch. I'll clean it up.

**Comment by squarewave** [comment]
Okay, assuming everything passes I'm ready for this to merge whenever.

**Pull request merged**